templates/userdash/userdash-top.html

```diff
@@ -1,13 +1,62 @@
 {{ define "content" }}
 
-<section class="jumbotron jumbotron-fluid text-center bg-dark">
-	<div class="container">
-		<h1 class="jumbotron-heading text-ca-yellow2"><strong>Dashboard</strong></h1>
+{{ template "basicspotlight" . }}
```

Comment:
@dpham273 Look at the html here and how i did it because it could be a good example for u